website/content/installation/_index.md

```diff
@@ -112,3 +112,23 @@ secretGenerator:
 generatorOptions:
  disableNameSuffixHash: true
 ```
+
+## Upgrade Current Version
```

Comment:
Just `Upgrade`?

Reply:
Yes, at the moment

Reply:
Sorry I wasn't clear, I meant this:
```suggestion
 ## Upgrade
```

Does it make sense? (i.e. just call the section "upgrade")